microsoft/iot-cardboard-js
# Error Handling for Scene Page

### Summary of changes 🔍 
>This PR mainly enabled the button action for json schema error. It calls the Copy Blob API call to copy the corrupted file and reset the 3DConfiguration file back to the default setting. I also, resolved minor PR suggestions from the previous PR about the scene page errors.


### Testing 🧪

1.   Corrupt your 3DConfiguration file on Azure Portal (Ex: delete an opening or closing bracket)
2.   Refresh your storybook and you should now see a json schema error occur
3.   Click on the "Reset Configuration File" button 
4.   Go back to your blob container, you should now see a 3DConfiguration file that has a default json schema and you should also see a new file titled "corrupted_date_timestamp"

### Checklist ✔️
